src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefreshOptions.cs

```diff
@@ -38,12 +38,17 @@ public AzureAppConfigurationRefreshOptions Register(string key, bool refreshAll)
         /// <param name="refreshAll">If true, a change in the value of this key refreshes all key-values being used by the configuration provider.</param>
         public AzureAppConfigurationRefreshOptions Register(string key, string label = LabelFilter.Null, bool refreshAll = false)
         {
-            RefreshRegistrations[key] = new KeyValueWatcher
+            if (string.IsNullOrEmpty(key))
```

Comment:
Guess we can put this in the release notes. Shouldn't effect anybody but technically a change.